packages/api/src/env.js

```diff
@@ -26,19 +26,16 @@ import pkg from '../package.json'
 export function envAll (_, env, event) {
   env.sentry = (env.SENTRY_DSN || typeof SENTRY_DSN !== 'undefined') && new Toucan({
     dsn: env.SENTRY_DSN || SENTRY_DSN,
-    event,
+    context: event,
```

Comment:
the`event` property is deprecated in toucan-js https://github.com/robertcepa/toucan-js/releases/tag/v2.5.0